x-pack/test/case_api_integration/basic/tests/cases/comments/delete_comment.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+
+import { CASES_URL } from '../../../../../../plugins/case/common/constants';
+import { postCaseReq, postCommentReq } from '../../../../common/lib/mock';
+import { deleteCases, deleteCasesUserActions, deleteComments } from '../../../../common/lib/utils';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
+  const supertest = getService('supertest');
+  const es = getService('legacyEs');
```

Comment:
Consider using `getService('es')` instead of `legacyEs`. @FrankHassanabad's [PR](https://github.com/elastic/kibana/pull/65598) fixed the client's issue.